k8s/postgresql.yaml

```diff
@@ -0,0 +1,38 @@
+# PostgreSQL StatefulSet
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgresql-db
+spec:
+  serviceName: postgresql-db-service
+  replicas: 3
+  selector:
+    matchLabels:
+      app: postgresql-db
+  template:
+    metadata:
+      labels:
+        app: postgresql-db
+    spec:
+      containers:
+        - name: postgresql-db
+          image: postgres:12.8
+          ports:
+            - containerPort: 5432
+              name: http
```

Comment:
PostgreSQL doesn't use HTTP.

Reply:
name: postgresdb
or just remove that line.

Not very sure tbh.

Reply:
Yes, I have added it to try the DNS name by using the port name. (Just a dummy name)
So I think it is better to be removed for now.